src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -31,18 +33,30 @@ class FeelingRoom extends Component {
     )
   }
   createAngryRoom = () => {
-    //functions to animate go here
-    return (
-      console.log('ANGRY')
-      //elements to animate go here
+    const squareNums = Array.from(Array(10).keys())
+      angryFeelingAnimation()
+    const angrySquares = squareNums.map(num => {
```

Comment:
I can not begin to articulate how much I love the angry squares. Your animation is GREAT, and really reflective of the vibe we hoped would be created in this room.

Reply:
I agree! The emotion of this room is great.